File: AirportApp.Domain/Entities/AircraftMember.cs
using System.ComponentModel.DataAnnotations;
using AirportApp.Domain.Entities.Common;
using AirportApp.Domain.Entities.Enums;

namespace AirportApp.Domain.Entities
{
    public class AircraftMember : EntityBase
    {
        [Required]
        public string Surname { get; set; }

        [Required]
        public string Name { get; set; }

        public string Patronymic { get; set; }

        [Required]
        public AircraftPosition Position { get; set; }

        [Required]
        public int AircraftId { get; set; }

        public Aircraft Aircraft { get; set; }
    }
}

File: AirportSpaApp/ClientApp/src/models/AircraftMember.ts
import { Aircraft } from "./Aircraft";

export interface AircraftMember {
    id: number;
    surname: string;
    name: string;
    patronymic: string;
    position: AircraftPosition;
    aircraftId: number;
    aircraft?: Aircraft;
}

enum AircraftPosition {
    Capitan,
    Pilot,
    Attendant
}

export interface AircraftMemberFilterModel {
    flightNumber: string;
}

export const aircraftPositionTranslates: any = {
    [AircraftPosition.Capitan]: 'Капитан',
    [AircraftPosition.Pilot]: 'Пилот',
    [AircraftPosition.Attendant]: 'Стюард',
}